smithy-swift-codegen/src/test/kotlin/AuthSchemeResolverGeneratorTests.kt

```diff
@@ -96,7 +96,7 @@ class AuthSchemeResolverGeneratorTests {
     }
 
     private fun setupTests(smithyFile: String, serviceShapeId: String): TestContext {
-        val context = TestContext.initContextFrom(smithyFile, serviceShapeId, MockHttpRestJsonProtocolGenerator()) { model ->
+        val context = TestContext.initContextFrom(smithyFile, serviceShapeId, MockHTTPRestJsonProtocolGenerator()) { model ->
```

Comment:
Tests are adapted to changes in protocol generator interfaces, and expected code is updated.  No further comments on codegen test files below.